app/models/submission_rule.rb

```diff
@@ -27,13 +27,13 @@ def get_collection_time
     return @get_collection_time if !@get_collection_time.nil?
     @get_collection_time = calculate_collection_time
   end
-
+  
   def calculate_collection_time
     assignment.latest_due_date + hours_sum.hours
   end
 
   def calculate_grouping_collection_time(grouping)
-    if grouping.inviter.section
+    if !grouping.inviter.nil?
```

Comment:
@MatakuT This change breaks existing functionality. SectionDueDate should only be queried when a grouping is part of a section.